core/DbConnection.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace app\core;
+
+class DbConnection
+{
+    public function connect(){
+        $mysqli = new \mysqli("localhost", "root", "", "vbis");
```

Comment:
Why \mysqli?

This is not good, use class mysqli and then use action re-review code.